libs/shared/src/features/audio-playback/audio-playback.component.html

```diff
@@ -0,0 +1,8 @@
+<button (click)="togglePlayback()" class="audio-button">
+  <div *ngIf="isAudioPlaying; else playIcon">
```

Comment:
nit(non-blocking) - it would be good for the icon to change when play button first clicked (currently only changes when clicked twice, once to start and once to pause), so that the user gets feedback that the audio click was registered. Could also consider putting the icon in a `mat-button` to get a ripple effect

nit(non-blocking) - feel free if you want to use the `@if(...)` and `@else(...)` syntax. Can also probably just put the if/else around the icon and not need additional ng-template